### PR TITLE
Update help-desk-operators.md

### PR DESCRIPTION
**Pull request by mackie1604**
Adding note for RBAC permission required to collect and access diagnostics in the troubleshooting blade

### CODE REVIEW

**Comment by Erikre** [comment]
#sign-off

File: memdocs/intune/fundamentals/help-desk-operators.md [hunk 1]
```diff
@@ -179,6 +179,9 @@ Enrollment restrictions are use to prevent (block) personally owned devices from
 
 Diagnostics for devices and applications.
 
+> [!NOTE]
+> To collect and access diagnostics you must have the collect diagnostics permission added to your role, see [Role-based administration control (RBAC) with Intune](role-based-access-control.md)
```
Comment (JohanFreelancer9) [inline memdocs/intune/fundamentals/help-desk-operators.md:183]:
```suggestion
> To collect and access diagnostics you must have the Collect diagnostics permission added to your role. For more information, see [Role-based administration control (RBAC) with Intune](role-based-access-control.md).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @mackie1604, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by Erikre** [comment]
#sign-off

**Pull request merged**